src/js/components/Notification/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export type StatusType = 'critical' | 'warning' | 'normal' | 'unknown';
+
+export interface NotificationProps {
+  title: string;
```

Comment:
Is title required or optional? 

If optional, then need `title?: string;`

If required, then NotificationType should make `title: PropTypes.string.isRequired,`